test/fw/ptl/utils/pbs_crayutils.py

```diff
@@ -176,14 +177,11 @@ def get_numthreads(self, nid):
                 numthreads = int(key[1])
         return numthreads
 
-    def num_compute_vnodes(self):
+    def num_compute_vnodes(self, server):
         """
         Count the Cray compute nodes and return the value.
         """
-        vlist = []
-        vnl = self.server.filter(
-            VNODE, {'resources_available.vntype': 'cray_compute'})
+        vnl = server.filter(MGR_OBJ_NODE,
+                            {'resources_available.vntype': 'cray_compute'})
         vlist = vnl["resources_available.vntype=cray_compute"]
-        nv = len(vlist)
-        self.assertNotEqual(nv, 0, "There are no cray_compute vnodes present.")
-        return nv
+        return len(vlist)
```

Comment:
Can you just put vnl[] in the len directly and get rid of vlist and still have the line under 79 characters?  Actually if you wanted to shrink the function even further, you could just return len(server.filter())

Reply:
len(server.filter()) would return 1. I will get rid of vlist by passing vnl[] to len().